exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,39 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of the same length are ok" do
+    assert {:ok, _} = DNA.hamming_distance('', '')
+  end
+
+  @tag :pending
+  test "hamming distance is undefined for strands of different lengths" do
```

Comment:
Consider making this the last test. We don't really need the error case until the happy path is fully driven out.